include/rsg/services/globalService.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef _RSG_GLOBAL_SERVICE_IMPL_
```

Comment:
Yes, please. I'd like the things in RSG to have the same names that in S4U, when possible.